magento/pwa-studio
# [PWA-2314]: Replace deprecated "code" and "root_category_id" from storeConfig queries

## Description

PR to replace any use of deprecated graphQL fields `code` and `root_category_id` on storeConfig queries. They were replaced by new fields `store_code` and `root_category_uid`.

## Related Issue







Closes PWA-2314.

## Acceptance





PWA Studio no longer requests the deprecated `code` and `root_category_id` fields on the `StoreConfig` type.

### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature




1. Verify that the homepage still functions correctly
2. Verify that navigating around the site works correctly
3. Verify that the store switcher still functions correctly
4. Test upgrade flow. So either on cloud or local hit all the changed area above in 12.1 (master) and then run staging server with this pr and check all areas in new tab of same browser. Details steps on upgrade test can be found here https://jira.corp.magento.com/browse/PWA-1418

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
